packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -82,6 +83,10 @@ export class SyncRepoSettings {
     const logger = this.logger;
     const repo = options.repo;
     const [owner, name] = repo.split('/');
+    if (!config?.enabled) {
```

Comment:
Looks like this should probably be
```suggestion
    if (config?.enabled === false) {
```

Then if the enabled field is not set, we default to running the sync.